README.md

```diff
@@ -136,6 +136,12 @@ apply to CloudFormation stacks but not EKS clusters.
 eksctl create cluster --tags environment=staging --region=us-east-1
 ```
 
+To configure node volume size, use the `--node-volume-size` flag.
+
+```
+eksctl create cluster --node-volume-size 50
```

Comment:
Please use `--node-volume-size=50`, for consistency with the rest of the readme.